packages/common/src/services/audius-backend/eagerLoadUtils.ts

```diff
@@ -3,6 +3,25 @@ import { LocalStorage } from '../local-storage'
 
 const DISCOVERY_PROVIDER_TIMESTAMP = '@audius/libs:discovery-node-timestamp'
 
+export const LIBS_INITTED_EVENT = 'LIBS_INITTED_EVENT'
+
+/**
+ * Wait for the `LIBS_INITTED_EVENT` or pass through if there
+ * already exists a mounted `window.audiusLibs` object.
+ */
+export const waitForLibsInit = async () => {
+  // If libs is already defined, it has already loaded & initted
+  // so do nothing
+  // @ts-ignore
+  if (window.audiusLibs) return
```

Comment:
I don't think this can be in `common` since this code also will run on mobile which doesn't have a `window`

Reply:
Hm how can I implement WaitForLibsInit in common then? Is there another way to access audiusLibs?

Reply:
Hmm i see, I think this is the reason that there was that indirection between wallet client and audius backend - the waitForLibs method gets injected into audius backend. I led you astray 😢

Reply:
Otherwise I can just revert to using AudiusBackend

Reply:
yeah I think back to backend makes sense sorry 😞